### DISCUSSION TITLE
I cant join any sessions. Always get the "Timed out while searching for a session" error.

### DISCUSSION THREAD

**Discussion by Rabtos** [YimMenu/YimMenu#2185] [General]
I think some modder f*ed my Account, I can't connect to GTA Online anymore. It always shows me the "Timed out while searching for a session" error.

I've read the only way is to ask another Modder to restore my Account, or ask Rockstar Support to reset my Character.
So I contacted the Rockstar Support and they said its a connection problem. I've tried my friends Account and could play.
Then they said they are aware of the problem and need to wait for them finding more Information. They didn't want to reset my Character..!

Is it possible to add Protection against this kinda stuff?
And can someone help me please?

EDIT: **Joining an SCTV Session, then finding a solo session with YimMenu worked out for me!**
But I have to redo the steps everytime I restart GTA.

EDIT2: I changed my character and tried joining a game after a fresh restart - without success. 
Used the SCTV Method and while doing it, 2 modders joined my lobby and it seems like they targeted only me. I kicked em out and since then can't join any lobbys again even trying the SCTV method. Now I'm really f-ed

EDIT3: A friend invited me to an Job, I accepted and got sent to an SCTV Lobby, wtf? Then I was bugged and could exit the Spectator Mode only with using "Leave GTA Online" with YimMenu

EDIT4: At unknowncheats someone wrote this: [QUOTE=whysus;3865531]Sometimes you can go to spectator slots, from there you can go to character selection and delete character. You applied "wrong" bools/ints and bricked it yourself.[/QUOTE]
**I think thats true, I tried changing some tattoos and I guess failed at it...** 

**EDIT5 (Solution): Just now I somehow managed to enter an SCTV lobby again and could go online, from there to character selection. Now it works again! I don't know how I did that though. Used STAND and activated random options trying to get it to work.**

**Comment by lonelybud** [comment]
damn this is concerning

**Comment by Yimura** [comment]
Have you considered just restarting your game? I seem to remember vaguely that there's a kick that messes with your game and doesn't allow you to join sessions until you restart.

As often is said in IT, "Have you tried turning it off and on?".

**Reply by Rabtos** [reply]
Yeah sure.
- Tried stand's "Unbrick Account" function. It didn't work.
- Going into "Character selection" gives me a blackscreen with an GTA Mouse. I have to Alt+F4 the game.
- I somehow managed to join a CCTV Lobby, where I could watch and chat with other players. 

Is there any other way to delete / reset my Characters ?

**Reply by Yimura** [reply]
Try removing the following folder `%USERPROFILE%\Documents\Rockstar Games\GTA V\Profiles` and see if that fixes it.

**Reply by Rabtos** [reply]
Tried it already. But I activated the Cloud Save, maybe it loaded the bricked data again?
Will try it now,.
EDIT:  No success :(

**Reply by Yimura** [reply]
Does enabling "Join SCTV slots" help you with joining online?

**Reply by Rabtos** [reply]
No :/ 
Edit: Is the "Join SCTV slots" option broken? 
I just connected to an Lobby using YimMenu's Session Switcher again. Any Idea what I could try to do from here? Could I use the Stat Editor and reset my Character with this?

**Comment by Rabtos** [comment]
At unknowncheats someone wrote this: [QUOTE=whysus;3865531]Sometimes you can go to spectator slots, from there you can go to character selection and delete character. You applied "wrong" bools/ints and bricked it yourself.[/QUOTE]
I think thats true, I tried changing some tattoos and I guess failed at it...

Edit: Just now I somehow managed to enter an SCTV lobby again and could go online, from there to character selection. Now it works again! I don't know how I did that though. Used STAND and activated random options trying to get it to work.
